File: python/roman_numerals_spec.py
import unittest
from roman_numerals import to_roman

class armstrong_test(unittest.TestCase):
    '''Test that the output is a list'''

    def test_returns_3(self):
        self.assertEqual(to_roman(3), 'III')
    def test_returns_4(self):
        self.assertEqual(to_roman(4), 'IV')
    def test_returns_43(self):
        self.assertEqual(to_roman(43), 'XLIII')
    def test_returns_70(self):
        self.assertEqual(to_roman(70), 'LXX')
    def test_returns_3724(self):
        self.assertEqual(to_roman(3724), 'MMMDCCXXIV')

if __name__ == '__main__':
    unittest.main()
# add tests to cover different edge cases

File: js/romanNumeralsSpec.js
var rn = require("./romanNumerals");

console.log(rn.toRoman(1) === 'I');
console.log(rn.toRoman(3) === 'III');
console.log(rn.toRoman(4) === 'IV');
console.log(rn.toRoman(9) === 'IX');
console.log(rn.toRoman(14) === 'XIV');
console.log(rn.toRoman(44) === 'XLIV');
console.log(rn.toRoman(944) === 'CMXLIV');
console.log(rn.toRoman(1000) === 'M');

console.log(rn.toRoman(2000) === 'MM'); 

